### PR TITLE
experimental: Enhance Performance by Using Postgrest to Fetch Data from Build Table

### PR DESCRIPTION
**Pull request by istarkov**
## Description

[Big Project Example](https://postgrest.development.webstudio.is/builder/be939bf2-9095-4e17-b175-69de2bbf9597)

Reduces time of big projects loading:

   ✓ { scenario:new_o }...........: avg=2.58s    min=1.99s  med=2.54s    max=3.53s    p(90)=2.95s    p(95)=3.15s    p(99)=3.44s    count=31
     ✓ { scenario:old_o }...........: avg=5.5s     min=4.55s  med=5.12s    max=15.58s   p(90)=5.44s    p(95)=6.02s    p(99)=12.87s   count=31

2 times faster. (but highly depends on test times, project size etc sometimes it's 6x difference)

Reduces patch time too.

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

**Review by kof — APPROVED** [review]

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1572,6 +1575,28 @@ importers:
         specifier: 5.5.2
         version: 5.5.2
 
+  packages/postgrest:
```
Comment (TrySound) [inline pnpm-lock.yaml:1578]: maybe call it db?

File: packages/postgrest/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "generate-types": "pnpx supabase gen types --lang=typescript --db-url postgresql://postgres:pass@localhost/webstudio > ./src/__generated__/db-types.ts"
+  },
+  "dependencies": {
+    "@supabase/postgrest-js": "^1.15.8"
+  },
+  "devDependencies": {
+    "@jest/globals": "^29.7.0",
+    "@webstudio-is/jest-config": "workspace:*",
+    "@webstudio-is/tsconfig": "workspace:*",
+    "type-fest": "^4.3.1",
```
Comment (TrySound) [inline packages/postgrest/package.json:28]: are jest and type-fest used?

Comment (istarkov) [reply packages/postgrest/package.json:28]: type-fest highly probably will be used based on their docs.
jest probably

File: pnpm-lock.yaml (hunk 1 above)
Comment (istarkov) [reply pnpm-lock.yaml:1578]: If we will start move onto then yes. Not a pig deal as it's a   "private": true package

File: packages/postgrest/package.json (hunk 2 above)
Comment (istarkov) [reply packages/postgrest/package.json:28]: Will remove jest

**Pull request merged**